.github/workflows/build.yaml

```diff
@@ -74,3 +68,25 @@ jobs:
         status: ${{ job.status }}
         channel: "#duos-notifications"
         fields: repo,commit,author,action,eventName,ref,workflow,job,took
+  report-to-sherlock:
+    if: github.event_name == 'push'
+      uses: broadinstitute/sherlock/.github/workflows/client-report-app-version.yaml@main
+      needs: [ tag-build-push ]
+      with:
+        new-version: ${{ needs.tag-build-push.outputs.sherlock-version }}
+        chart-name: 'duos'
```

Comment:
Chart name check.  I _think_ it's **consent**

Reply:
you are correct - updated this PR as well as the ontology one with "consent" and "ontology" chart names